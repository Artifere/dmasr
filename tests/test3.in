MAKE NC 8
ADD NC $1,$0
MAKE NC 1
SHL NC $0,$1
SHL NC $0,$1
SHL NC $0,$1
SHL NC $0,$1
SETSR NC $2
SHR NC $0,$1
SHL NC $0,$1

#Ici on teste le décalage gauche, ainsi que la bonne gestion des drapeaux suite à un SHL