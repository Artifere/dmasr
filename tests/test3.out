OOOOÏWO